bigtable/client/metadata_update_policy_test.cc

```diff
@@ -13,13 +13,278 @@
 // limitations under the License.
 
 #include "bigtable/client/metadata_update_policy.h"
+#include "bigtable/client/admin_client.h"
+#include "bigtable/client/internal/make_unique.h"
+#include "bigtable/client/table.h"
+#include "bigtable/client/table_admin.h"
+#include "google/bigtable/v2/bigtable.grpc.pb.h"
+#include <google/bigtable/admin/v2/bigtable_table_admin.grpc.pb.h>
 #include <gtest/gtest.h>
+#include <thread>
 
 std::string const kInstanceName = "projects/foo-project/instances/bar-instance";
 std::string const kTableId = "baz-table";
 std::string const kTableName =
     "projects/foo-project/instances/bar-instance/tables/baz-table";
 
+namespace btproto = google::bigtable::v2;
+namespace adminproto = google::bigtable::admin::v2;
+
+/**
```

Comment:
Please put these classes in an anonymous namespace.  Mostly for documentation purposes.